clouds/cli/cli.py

```diff
@@ -0,0 +1,106 @@
+import os
+import click
+import shutil
+from simple_logger.logger import get_logger
+from clouds.cli.aws.aws_cli import clean_aws_resources
+from pyhelper_utils.runners import function_runner_with_pdb
+from clouds.cli.microsoft_azure.microsoft_azure_cli import nuke_all_azure_resources
+from clouds.aws.aws_utils import set_and_verify_aws_credentials, aws_region_names
+
+LOGGER = get_logger(name="nuke-cli")
+
+
+@click.group()
+def aws() -> None:
+    pass
+
+
+@aws.command()
+@click.option(
+    "--aws-regions",
+    help="""
+        \b
+        Comma-separated string of AWS regions to delete resources from.
+        If not provided will iterate over all regions.
```

Comment:
`If not provided will iterate over all regions.` this is not True, this true only if `--all-aws-regions` passed

Reply:
done